lib/shared/bucketing-assembly-script/index-worker.js

```diff
@@ -0,0 +1,7 @@
+const { instantiate } = require('./build/bucketing-lib.worker')
+
+// Instantiation method for CF Worker / Web Worker environments
+exports.instantiate = async () => {
+    // eslint-disable-next-line no-undef
+    return await instantiate(BUCKETING_WASM, {})
```

Comment:
whats `BUCKETING_WASM`?

Reply:
its how we bind the WASM package using CF Workers: https://github.com/taplytics/DevCycle/pull/1041/files#diff-99e08ddf6951419c948852c037cf2961c0c05857906e6462f8a47cfa2fe20507R16